src/main/java/com/gooddata/GoodData.java

```diff
@@ -52,9 +52,9 @@
 
     public static final String GDC_REQUEST_ID_HEADER = "X-GDC-REQUEST";
 
-    private static final String PROTOCOL = "https";
-    private static final int PORT = 443;
-    private static final String HOSTNAME = "secure.gooddata.com";
+    protected static final String PROTOCOL = "https";
```

Comment:
Of course, I have - anytime you need to extend GoodData java you are forced to copy the default values: https://github.com/gooddata/msf-tools/blob/perf-tests/dl-tool/src/main/java/com/gooddata/msf/dltool/service/DlGoodData.java#L19